client/src/pages/course/mentor/confirm.tsx

```diff
@@ -31,21 +33,22 @@ function Page(props: { session: Session; courseAlias?: string }) {
   useAsync(async () => {
     try {
       setLoading(true);
-      const courseAlias = router.query?.course;
-      if (!courseAlias) {
+      const courseAlias = router.query?.course as string;
+      if (courseAlias == null) {
```

Comment:
Why not `===`?

Reply:
because it might not be. This check is needed in order to check for its existence. Why apply strict equality here, what problem will it solve or eliminate?